doc/de/weechat_quickstart.de.adoc

```diff
@@ -388,28 +410,42 @@ Entfernen der Tastenbelegung:
 [[plugins_scripts]]
 == Erweiterungen/Skripten
 
-Bei einigen Distributionen wie z.B. Debian, sind die Erweiterungen über separate Pakete
-erhältlich (z.B. weechat-plugins).
-Erweiterungen werden -sofern welche gefunden worden sind- automatisch geladen (Bitte beachte
-die Dokumentation zum installieren/entfernen von Erweiterungen und/oder Skripten).
+Bei einigen Distributionen (z.B. Debian),
+sind die WeeChat-Erweiterungen über separate Pakete erhältlich (z.B. _weechat-plugins_).
+Erweiterungen werden, insofern sie beim Start von WeeChat gefunden wurden,
+automatisch geladen (Bitte beachte die Dokumentation zum Installieren
+und Entfernen von Erweiterungen und/oder Skripten).
 
-Viele externe Skripten (von Drittprogrammierern) sind für WeeChat verfügbar. Diese
-Skripten können mittels des `/script` Befehls heruntergeladen und installiert werden:
+Viele externe Skripten (von Drittprogrammierern) sind für WeeChat verfügbar.
+Diese Skripten können mit Hilfe des `/script`-Befehls heruntergeladen
+und installiert werden:
 
 ----
 /script install go.py
 ----
 
-siehe `/help script` für weitere Informationen.
+Zeige weitere Informationen:
+----
+/help script
+----
+
+Zeige eine liste aller verfügbaren Skripten:
+----
+/script
+----
+
+Auf *weechat.org* von Drittanbietern bereitgestellte Scripten:
 
-Eine Liste aller verfügbaren Skripten kann man sich in WeeChat mittels
-`/script` anzeigen lassen oder man besucht folgende Webseite:
 https://weechat.org/scripts
```

Comment:
If you add a blank line above the URL, it makes a new paragraph instead of displaying the link inline, in the sentence.